Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -152,6 +152,11 @@ private bool IsInPrivateAddressSpace(string endpoint, bool checkSubnets)
                 return true;
             }
 
+            if (!IPAddress.TryParse(endpoint, out _))
+            {
+                return false;
+            }
+
             byte[] octet = IPAddress.Parse(endpoint).GetAddressBytes();
```

Comment:
```suggestion
            if (!IPAddress.TryParse(endpoint, out var ipAddress))
            {
                return false;
            }

            byte[] octet = ipAddress.GetAddressBytes();
```

Don't need to parse the ip twice.

Reply:
@Bond-009 Sorry, thought resolved was accept change.